addons/partner_autocomplete_bridge_address_extended/__manifest__.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
```

Comment:
`partner_autocomplete_address_extended` is shorter and nicer than `partner_autocomplete_bridge_address_extended`